test/mpi/dtpools/include/dtpools_internal.h

```diff
@@ -121,9 +121,94 @@ typedef enum {
     DTPI_OBJ_TYPE__SUBARRAY_C,
     DTPI_OBJ_TYPE__SUBARRAY_F,
     DTPI_OBJ_TYPE__STRUCT,
+/*
+    DTPI_OBJ_TYPE__STRUCT_BASIC,
+    DTPI_OBJ_TYPE__STRUCT_CONTIG,
+    DTPI_OBJ_TYPE__STRUCT_VECTOR,
+    DTPI_OBJ_TYPE__STRUCT_HVECTOR,
+    DTPI_OBJ_TYPE__STRUCT_BLOCK_INDEXED,
+    DTPI_OBJ_TYPE__STRUCT_BLOCK_HINDEXED,
+    DTPI_OBJ_TYPE__STRUCT_INDEXED,
+    DTPI_OBJ_TYPE__STRUCT_HINDEXED,
+    DTPI_OBJ_TYPE__STRUCT_SUBARRAY_C,
+    DTPI_OBJ_TYPE__STRUCT_SUBARRAY_F,
+*/
```

Comment:
What's the reason for having these commented names?

Reply:
They are just suggestion on possible future expansions for the struct types. I can remove them.

Reply:
Suggestion for the future is OK as long as the comment indicates that is the case. Right now the reader does not know.

Reply:
Ok will keep that in mind for next time. Thanks